README.md

```diff
@@ -73,9 +73,33 @@ modules:
   icmp:
     prober: icmp
     timeout: 5s
+  dns_udp:
+    prober: dns
+    timeout: 5s
+    dns:
+      query_name: "www.prometheus.io"
+      query_type: "A"
+      valid_rcodes:
+      - 0
```

Comment:
Can these be the names rather than numbers? I expect most people don't know the code for NXDOMAIN.